### ISSUE TITLE
Input focus on presentation after opening it

### ISSUE THREAD

**Issue by chbndrhnns** [gitpitch/gitpitch#47]
The keyboard focus should be on the presentation so that it is no longer necessary to click into the presentation to be able to use the shortcuts.

**Comment by gitpitch** [comment]
Input/keyboard focus now set automatically when presentation loads. Resolved by 030fc8e76500ade172c838e9e15e20f1799ffaed.
